### PR TITLE
reference/best-practices: update HAProxy doc

### PR DESCRIPTION
**Pull request by juliezhang1112**
### What is changed, added or deleted? 
- Fix format issues: `2.6/3.x` and `8/9` are misinterpreted as fractions
- Improve the wording: `OpenBSD 3.1 到最新版操作系统` -> `OpenBSD 3.1 及以上版本操作系统`

### What is the related PR or file link(s)? 
https://github.com/pingcap/docs-cn/pull/1889

### Which version does your change affect? 
dev, v3.0, v2.1

### CODE REVIEW

**Comment by juliezhang1112** [comment]
PTAL @lilin90

File: dev/reference/best-practices/haproxy.md [hunk 1]
```diff
@@ -44,11 +44,11 @@ HAProxy 由 Linux 内核的核心贡献者 Willy Tarreau 于 2000 年编写，
...
 
 - Linux 2.4 操作系统，支持 x86、x86_64、Alpha、Sparc、MIPS 和 PARISC 架构。
-- Linux 2.6/3.x 操作系统，支持 x86、x86_64、ARM、Sparc 和 PPC64 架构。
-- Solaris 8/9 操作系统，支持 UltraSPARC 2/3 架构。
+- Linux 2.6 或 3.x 操作系统，支持 x86、x86_64、ARM、Sparc 和 PPC64 架构。
+- Solaris 8 或 9 操作系统，支持 UltraSPARC 2/3 架构。
 - Solaris 10 操作系统，支持 Opteron/UltraSPARC 架构。
 - FreeBSD 4.10~10 操作系统，支持 x86 架构。
-- OpenBSD 3.1 到最新版操作系统，支持 i386、amd64、macppc、alpha 和 sparc64 架构。
+- OpenBSD 3.1 及以上版本操作系统，支持 i386、amd64、macppc、alpha 和 sparc64 架构。
```
Comment (lilin90) [inline dev/reference/best-practices/haproxy.md]:
```suggestion
- OpenBSD 3.1 及以上版本操作系统，支持 i386、AMD64、macppc、Alpha 和 SPARC64 架构。
```

File: dev/reference/best-practices/haproxy.md [hunk 2]
```diff
@@ -44,11 +44,11 @@ HAProxy 由 Linux 内核的核心贡献者 Willy Tarreau 于 2000 年编写，
 #### 操作系统
 
 - Linux 2.4 操作系统，支持 x86、x86_64、Alpha、Sparc、MIPS 和 PARISC 架构。
```
Comment (lilin90) [inline dev/reference/best-practices/haproxy.md]:
```suggestion
- Linux 2.4 操作系统，支持 x86、x86_64、Alpha、SPARC、MIPS 和 PA-RISC 架构。
```

File: dev/reference/best-practices/haproxy.md [hunk 3]
```diff
@@ -44,11 +44,11 @@ HAProxy 由 Linux 内核的核心贡献者 Willy Tarreau 于 2000 年编写，
...
-- Linux 2.6/3.x 操作系统，支持 x86、x86_64、ARM、Sparc 和 PPC64 架构。
-- Solaris 8/9 操作系统，支持 UltraSPARC 2/3 架构。
+- Linux 2.6 或 3.x 操作系统，支持 x86、x86_64、ARM、Sparc 和 PPC64 架构。
```
Comment (lilin90) [inline dev/reference/best-practices/haproxy.md]:
```suggestion
- Linux 2.6 或 3.x 操作系统，支持 x86、x86_64、ARM、SPARC 和 PPC64 架构。
```

File: dev/reference/best-practices/haproxy.md [hunk 4]
```diff
@@ -44,11 +44,11 @@ HAProxy 由 Linux 内核的核心贡献者 Willy Tarreau 于 2000 年编写，
...
+- Solaris 8 或 9 操作系统，支持 UltraSPARC 2/3 架构。
```
Comment (lilin90) [inline dev/reference/best-practices/haproxy.md]:
```suggestion
- Solaris 8 或 9 操作系统，支持 UltraSPARC II 和 UltraSPARC III 架构。
```

File: dev/reference/best-practices/haproxy.md [hunk 5]
```diff
@@ -43,12 +43,12 @@ HAProxy 由 Linux 内核的核心贡献者 Willy Tarreau 于 2000 年编写，
...
-- Linux 2.6/3.x 操作系统，支持 x86、x86_64、ARM、Sparc 和 PPC64 架构。
-- Solaris 8/9 操作系统，支持 UltraSPARC 2/3 架构。
-- Solaris 10 操作系统，支持 Opteron/UltraSPARC 架构。
+- Linux 2.4 操作系统，支持 x86、x86_64、Alpha、SPARC、MIPS 和 PA-RISC 架构。
+- Linux 2.6 或 3.x 操作系统，支持 x86、x86_64、ARM、SPARC 和 PPC64 架构。
+- Solaris 8 或 9 操作系统，支持 UltraSPARC II 和 UltraSPARC III 架构。
+- Solaris 10 操作系统，支持 Opteron 和 UltraSPARC 架构。
 - FreeBSD 4.10~10 操作系统，支持 x86 架构。
-- OpenBSD 3.1 到最新版操作系统，支持 i386、amd64、macppc、alpha 和 sparc64 架构。
+- OpenBSD 3.1 及以上版本操作系统，支持 i386、AMDd64、macppc、Alpha 和 SPARC64 架构。
```
Comment (lilin90) [inline dev/reference/best-practices/haproxy.md]:
```suggestion
- OpenBSD 3.1 及以上版本操作系统，支持 i386、AMD64、macppc、Alpha 和 SPARC64 架构。
```

File: v2.1/reference/best-practices/haproxy.md [hunk 6]
```diff
@@ -43,12 +43,12 @@ HAProxy 由 Linux 内核的核心贡献者 Willy Tarreau 于 2000 年编写，
...
-- Linux 2.6/3.x 操作系统，支持 x86、x86_64、ARM、Sparc 和 PPC64 架构。
-- Solaris 8/9 操作系统，支持 UltraSPARC 2/3 架构。
-- Solaris 10 操作系统，支持 Opteron/UltraSPARC 架构。
+- Linux 2.4 操作系统，支持 x86、x86_64、Alpha、SPARC、MIPS 和 PA-RISC 架构。
+- Linux 2.6 或 3.x 操作系统，支持 x86、x86_64、ARM、SPARC 和 PPC64 架构。
+- Solaris 8 或 9 操作系统，支持 UltraSPARC II 和 UltraSPARC III 架构。
+- Solaris 10 操作系统，支持 Opteron 和 UltraSPARC 架构。
 - FreeBSD 4.10~10 操作系统，支持 x86 架构。
-- OpenBSD 3.1 到最新版操作系统，支持 i386、amd64、macppc、alpha 和 sparc64 架构。
+- OpenBSD 3.1 及以上版本操作系统，支持 i386、AMDd64、macppc、Alpha 和 SPARC64 架构。
```
Comment (lilin90) [inline v2.1/reference/best-practices/haproxy.md]:
```suggestion
- OpenBSD 3.1 及以上版本操作系统，支持 i386、AMD64、macppc、Alpha 和 SPARC64 架构。
```

File: v3.0/reference/best-practices/haproxy.md [hunk 7]
```diff
@@ -43,12 +43,12 @@ HAProxy 由 Linux 内核的核心贡献者 Willy Tarreau 于 2000 年编写，
...
-- Linux 2.6/3.x 操作系统，支持 x86、x86_64、ARM、Sparc 和 PPC64 架构。
-- Solaris 8/9 操作系统，支持 UltraSPARC 2/3 架构。
-- Solaris 10 操作系统，支持 Opteron/UltraSPARC 架构。
+- Linux 2.4 操作系统，支持 x86、x86_64、Alpha、SPARC、MIPS 和 PA-RISC 架构。
+- Linux 2.6 或 3.x 操作系统，支持 x86、x86_64、ARM、SPARC 和 PPC64 架构。
+- Solaris 8 或 9 操作系统，支持 UltraSPARC II 和 UltraSPARC III 架构。
+- Solaris 10 操作系统，支持 Opteron 和 UltraSPARC 架构。
 - FreeBSD 4.10~10 操作系统，支持 x86 架构。
-- OpenBSD 3.1 到最新版操作系统，支持 i386、amd64、macppc、alpha 和 sparc64 架构。
+- OpenBSD 3.1 及以上版本操作系统，支持 i386、AMDd64、macppc、Alpha 和 SPARC64 架构。
```
Comment (lilin90) [inline v3.0/reference/best-practices/haproxy.md]:
```suggestion
- OpenBSD 3.1 及以上版本操作系统，支持 i386、AMD64、macppc、Alpha 和 SPARC64 架构。
```

**Comment by lilin90** [comment]
@juliezhang1112 Please do not directly create braches in the upstream repo.

**Comment by lilin90** [comment]
@juliezhang1112 
- Please address my new comments.
- I suggest using the `pending-aligning` label because when you align docs with #1889, you also need to refer to this PR.

**Comment by juliezhang1112** [comment]
> @juliezhang1112
> 
> * Please address my new comments.
> * I suggest using the `pending-aligning` label because when you align docs with #1889, you also need to refer to this PR.

Got it!

**Comment by superlzs0476** [comment]
LGTM

**Pull request merged**